copy-to-gcloud.sh

```diff
@@ -24,40 +24,77 @@ echo "LOCAL_DIRNAME=$LOCAL_DIRNAME"
 echo "GCLOUD_PATH=$GCLOUD_PATH"
 
 
+function copy_path_if_exists {
+    # Two arguments
+    # 1) local path to file
+    # 2) remote name of file relative to GCLOUD_PATH
+    SOURCE="$1"
+    DEST="gs://$GCLOUD_PATH/$2"
+    if [[ -f "$SOURCE" ]]; then
+        gsutil -m cp "$SOURCE" "$DEST"
+     else
+        echo "Skipping $SOURCE, file not found"
+    fi
+}
+
+function copy_local_file_if_exists {
```

Comment:
this function expects a basename, maybe rename to `copy_basename_if_exists`? it's kinda confusing to have two functions named "copy_local_file_if_exists" and "copy_path_if_exists", the names are too similar to be descriptive of the differences